gemfiles/rails_5_2.gemfile

```diff
@@ -4,13 +4,13 @@ source "https://rubygems.org"
 
 gem "rails", "~> 5.2"
 gem "railties", "~> 5.2"
-gem "capybara", "~> 3.0"
+gem "capybara", "~> 3"
```

Comment:
This might need a slight prod.

Reply:
What do you mean? Do I need to change anything?